src/clr-angular/button/button-loading/loading-button.module.ts

```diff
@@ -7,15 +7,16 @@
 import {CommonModule} from "@angular/common";
 import {NgModule, Type} from "@angular/core";
 
-import {ClrLoadingModule} from "../../utils/loading/loading.module";
+import {ClrIconModule} from "../../icon";
+
 import {ClrLoadingButton} from "./loading-button";
 
 export const CLR_LOADING_BUTTON_DIRECTIVES: Type<any>[] = [ClrLoadingButton];
 
 @NgModule({
-    imports: [CommonModule, ClrLoadingModule],
+    imports: [CommonModule, ClrIconModule],
```

Comment:
Why do you need `ClrIconModule`?

Reply:
Oh this was when I was testing out using clr-icon for the spinner graphics. Will remove.